.travis.yml

```diff
@@ -1,47 +1,46 @@
 language: java
-sudo: required
-dist: trusty
+dist: xenial
```

Comment:
The EOL of this `trusty` (Ubuntu 14.04) is coming, then migrate to `xenial` (Ubuntu 16.04).
https://wiki.ubuntu.com/Releases